aries-site/src/examples/templates/dashboards/content/chartCards/CostCompact.js

```diff
@@ -0,0 +1,179 @@
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+  const chartColors = useMemo(
+    () =>
+      Object.entries(theme.global.colors)
+        .filter(([key]) => key.includes('graph'))
+        .map(([, value]) => value),
+    [theme.global.colors],
+  );
+
+  // Set reporting window
+  useEffect(() => {
+    setReportWindow(prevWindow => ({
+      ...prevWindow,
+      begin: new Date(
+        new Date().setDate(
+          prevWindow.end.getDate() - REPORT_WINDOW_MAP[period],
+        ),
+      ),
+    }));
+  }, [period]);
+
+  // Filter and assemble data set
+  useEffect(() => {
+    if (consumptionData) {
+      const nextValues = [];
+      consumptionData.forEach(datum => {
+        const dataPointDate = new Date(datum.endDate);
+        if (
+          dataPointDate > reportWindow.begin &&
+          dataPointDate < reportWindow.end
+        ) {
+          const key = datum.service;
+          const value = parseFloat(datum.cost.replace(/[$,]/gm, ''));
+          const index = nextValues.findIndex(el => el.key === key);
+
+          if (index >= 0) {
+            const nextValue = nextValues[index].value + value;
+            nextValues[index].value = nextValue;
+            nextValues[index].displayValue = formatCurrency(nextValue);
+          } else {
+            nextValues.push({
+              key,
+              label: `${datum.service[0].toLocaleUpperCase()}${datum.service
+                .slice(1)
+                .toLocaleLowerCase()}`,
+              value,
+              displayValue: formatCurrency(value),
+              color: undefined,
+            });
+          }
+        }
+      });
+      nextValues
+        .sort((a, b) => b.value - a.value)
+        .forEach((value, index) => {
+          // eslint-disable-next-line no-param-reassign
+          value.color = chartColors[index % chartColors.length];
+        });
+      setValues(nextValues);
+    }
+  }, [chartColors, consumptionData, reportWindow.begin, reportWindow.end]);
+
+  // Calculate total cost
+  useEffect(() => {
+    let nextTotalCost = 0;
+    if (values) {
+      Object.keys(values).forEach(key => {
+        nextTotalCost += values[key].value;
+      });
+      setTotalCost(nextTotalCost);
+    }
+  }, [values]);
+
+  const grid = {
+    columns: {
+      xsmall: [['xsmall', 'small'], 'auto'],
+      small: [['xsmall', 'small'], 'auto'],
+      medium: ['auto', 'auto'],
+      large: ['auto', 'auto'],
+      xlarge: ['auto', 'auto'],
+    },
+    rows: [['auto']],
+    areas: {
+      xsmall: [['chart', 'legend']],
+      small: [['chart', 'legend']],
+      medium: [['chart', 'legend']],
+      large: [['chart', 'legend']],
+      xlarge: [['chart', 'legend']],
+    },
+    gap: {
+      xsmall: 'medium',
+      small: 'medium',
+      medium: 'small',
+      large: 'medium',
+      xlarge: 'medium',
+    },
+  };
+
+  return (
+    <ChartCard title="Cost" subtitle="Consumption Analytics">
```

Comment:
are we not adding the icons as part of the title on the right side ?

Reply:
At the moment we are not since we don't have any guidance/research are what those buttons would do.

Reply:
got it good to know! thanks for clarifying